For debugging, we probably want to disable method caching. Dunno

diff --git a/vm_opts.h b/vm_opts.h
index 80a00ee..6c4082c 100644
--- a/vm_opts.h
+++ b/vm_opts.h
@@ -36,8 +36,8 @@

 /* VM running option */
 #define OPT_CHECKED_RUN              1
-#define OPT_INLINE_METHOD_CACHE      1
-#define OPT_GLOBAL_METHOD_CACHE      1
+#define OPT_INLINE_METHOD_CACHE      0
+#define OPT_GLOBAL_METHOD_CACHE      0
 #define OPT_BLOCKINLINING            0

 /* architecture independent, affects generated code */
